### PR TITLE
Bug 1920159: Adjust CPU request for controller manager more precisely

### PR DESCRIPTION
**Pull request by smarterclayton**
The actual use of the KCM in large production environments is
slightly lower than the request, and is fairly constant. In order
to ensure more headroom is available on small node sizes, tweak
downwards closer to the observed 40m real usage while still leaving
some headroom.

/hold

For testing with the other related changes

### CODE REVIEW

**Comment by smarterclayton** [comment]
/retest

Not related to the change

/hold cancel

Combined tests looked ok.  Team should review and see the comments in the bug.

**Review by soltysh — APPROVED** [review]
/lgtm

**Comment by smarterclayton** [comment]
This repo's unit test jobs look broken.  I think this is a job that needs to use $ARTIFACT_DIR as steve noted in his aos-devel email.

**Comment by smarterclayton** [comment]
Testing that theory in https://github.com/openshift/release/pull/15420

**Comment by smarterclayton** [comment]
/retest

now that it is fixed

**Pull request merged**
